### DISCUSSION TITLE
Add possibility to clear text input fields or do it automatically with new type

### DISCUSSION THREAD

**Discussion by MasterSergius** [zauberzeug/nicegui#3747] [Ideas / Feature Requests]
Current logic adds text with each new type when I use user fixture in testing like the following:

```py
user.find("my_input").type("text1")  # contains "text1"
user.find("my_input").type("text2")  # contains "text1text2"
```

It would be good to have a clear method, or flag in type or whatever...

**Comment by rodja** [comment]
Thanks for bringing this up @MasterSergius! You can do this right now with `user.find('my_input').elements.pop().set_value('')`. But it would be great if you could call `user.find(...).clear()` to empty every `ValueElement`. I'll prepare a pull request.

**Reply by rodja** [reply]
PR is created as #3750.

**Reply by MasterSergius** [reply]
cool
Imho, pop is not safe here, so I use the following workaround so far:

```py
input_interaction = user.find("my_input")
for elem in input_interaction.elements:
   elem.value = ""
input_interaction.type("text2")
```

**Comment by falkoschindler** [comment]
PR #3750 has just been released. 🚀
